docs/content/misc/faq.ngdoc

```diff
@@ -130,6 +130,15 @@ AngularJS 1.3 only supports jQuery 2.1 or above. jQuery 1.7 and newer might work
 but we don't guarantee that.
 
 
+### When will the deprecated features be removed from the library?
+Most of the time we remove a deprecated feature in a next minor bump. For example, the `preAssignBindingsEnabled`
+`$compileProvider` method was defined in AngularJS `1.5.10`, deprecated in `1.6` and will be removed in `1.7`.
```

Comment:
Hmm, maybe it'd be better to put sth else here, preferably something where we adhered to this strategy. :)